x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -169,6 +177,45 @@ export function TimeseriesChart({
             />
           );
         })}
+
+        {anomalySeries &&
+          anomalySeries.map((anomalySerie) => {
+            if (anomalySerie.type === 'area') {
```

Comment:
Do we render the anomaly series as a line chart somewhere?

Reply:
No, only here.

Reply:
If you're asking this because you think I should create a new Component for this, I can't, at least until this issue is fixed on elastic chart. https://github.com/elastic/elastic-charts/issues/914

Reply:
I'm asking because I don't understand why we have the conditional. What would happen if you removed the `if`?

Reply:
The condition is needed because we have two types of charts to load, `AreasSeries` for anomaly boundaries, and `RectAnnotation` for anomaly score.

Reply:
If there is always only two series in

Reply:
If the number of series in `anomalySeries` is fixed, I think it would be better if we identify them by names than properties and store them in a map. So instead of:

```ts
anomalySeries?: TimeSeries[];
```

It should be something like:
```ts
anomalySeries?: {
  anomalyBoundaries: TimeSeries[];
  anomalyScores: TimeSeries[];
}
```

Reply:
Btw. looking at this code it looks like we never want to render both the boundaries and the scores at the same time. Is that correct?

Reply:
>Btw. looking at this code it looks like we never want to render both the boundaries and the scores at the same time. Is that correct?

It'll load both charts like:
<img width="689" alt="Screenshot 2020-11-26 at 13 59 15" src="https://user-images.githubusercontent.com/55978943/100607498-504fdc80-330b-11eb-9821-777643436d3d.png">